src/inspector_agent.cc

```diff
@@ -7,19 +7,15 @@
 #include "node_crypto.h"
 #include "node_mutex.h"
 #include "node_version.h"
+#include "v8-inspector.h"
 #include "v8-platform.h"
 #include "util.h"
 #include "zlib.h"
 
-#include "platform/v8_inspector/public/InspectorVersion.h"
-#include "platform/v8_inspector/public/V8Inspector.h"
-#include "platform/v8_inspector/public/V8InspectorClient.h"
-#include "platform/v8_inspector/public/V8InspectorSession.h"
-#include "platform/v8_inspector/public/V8StackTrace.h"
-#include "platform/inspector_protocol/InspectorProtocol.h"
-
 #include "libplatform/libplatform.h"
 
+#include <unicode/unistr.h>
```

Comment:
If you're taking a dependency on ICU, then configure should have a `--with-intl=none` -> `--without-inspector` implication.